stage1/init/common/pod.go

```diff
@@ -256,28 +259,61 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		return errwrap.Wrap(errors.New("unable to write environment file"), err)
 	}
 
-	// This is a partial implementation for app.User and app.Group:
-	// For now, only numeric ids (and the string "root") are supported.
-	var uid, gid int
+	var _uid, gid int
```

Comment:
Oh, I missed that. Ok, let's keep it like that then. I'm not aware of any rules for that.